test/unit/github/cli/test_ci_get_build_runner.py

```diff
@@ -0,0 +1,45 @@
+from test.unit.github.cli.cli_runner import CliRunner
+from unittest.mock import MagicMock
+
+import pytest
+from _pytest.monkeypatch import MonkeyPatch
+
+import exasol.slc_ci.lib.get_build_runner as lib_get_build_runner
+from exasol.slc_ci.cli.commands.get_build_runner import get_build_runner
+from exasol.slc_ci.lib.github_access import GithubAccess
+
+
+@pytest.fixture
+def cli():
+    return CliRunner(get_build_runner)
+
+
+@pytest.fixture
+def mock_get_build_runner(monkeypatch: MonkeyPatch) -> MagicMock:
+    mock_function_to_mock = MagicMock()
+    monkeypatch.setattr(lib_get_build_runner, "get_build_runner", mock_function_to_mock)
+    return mock_function_to_mock
+
+
+def test_get_build_runner_no_flavor(cli):
+    assert cli.run().failed and "Missing option '--flavor'" in cli.output
+
+
+def test_get_build_runner_no_github_var(cli):
+    assert (
+        cli.run("--flavor", "abc").failed
+        and "Missing option '--github-var'" in cli.output
+    )
+
+
+def test_get_build_runner(cli, mock_get_build_runner):
+    cli.run("--flavor", "flavor_a", "--github-var", "abc")
+    assert cli.succeeded
```

Comment:
Why that complicated 

With assert mock_calls == [call(flavor=...,] you can handle all the asserts in one assert and probably also get better diff in case of a bug.

Ah, I see you need to test the GitHubAccess, I think that is best done by a Matcher Object 

mock_calls == [call(flavor=...,github_access=IsInstance(GithubAccess))]